velox/docs/functions/spark/aggregate.rst

```diff
@@ -107,13 +107,17 @@ General Aggregate Functions
 
     Returns the sum of `x`.
 
-    Supported types are TINYINT, SMALLINT, INTEGER, BIGINT, REAL and DOUBLE.
+    Supported types are TINYINT, SMALLINT, INTEGER, BIGINT, REAL, DOUBLE and DECIMAL.
 
     When x is of type DOUBLE, the result type is DOUBLE.
     When x is of type REAL, the result type is REAL.
+    When x is of type DECIMAL(p, s), the result type is DECIMAL(p + 10, s), where (p + 10) is capped at 38.
+
     For all other input types, the result type is BIGINT.
 
-    Note: When the sum of BIGINT values exceeds its limit, it cycles to the overflowed value rather than raising an error.
+    Note:
+    For DECIMAL type, when an overflow occurs in the accumulation, it returns null. For all other input types,
+    when the sum of input values exceeds its limit, it cycles to the overflowed value rather than raising an error.
```

Comment:
> it cycles to the overflowed value

I assume this behavior doesn't apply to REAL and DOUBLE types. If so, then saying "For all other input types" is incorrect. Would you double check?

Reply:
Actually, this explanation comes from Spark's comments.

https://github.com/apache/spark/blob/f5b0de07eff49bc1d076c4a1dc59c8672beff99e/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/Sum.scala#L129

But you are right, Spark's own documentation is not entirely accurate. Based on my tests, when an overflow occurs with double and real, the result is Infinity, not an overflowed value.  I will fix the documentation,.
```
spark-sql> SELECT sum(col) FROM VALUES (1E308), (1E308) AS tab(col);
Infinity
```